pypesto/result/optimize.py

```diff
@@ -162,6 +181,25 @@ def __getitem__(self, index):
     def __len__(self):
         return len(self.list)
 
+    def summary(self):
+        """Get summary of the object."""
+        # perform clustering for better information
+        clust, clustsize = assign_clusters(delete_nan_inf(self.fval)[1])
+
+        message = (
+            "## Optimization Result \n\n"
+            f"* number of starts: {len(self)} \n"
+            f"* best value found {clustsize[0]} time(s) \n"
```

Comment:
Since it's with some tolerance
```suggestion
            f"* best value (approximately) found {int(clustsize[0])} time(s) \n"
```